packages/contracts-bedrock/scripts/print-addresses.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Default directory base path
+BASE_DIR="./deployments"
+
+# Directory name from the argument
+DIR_NAME=$1
+
+# Check for the --sdk flag
+SDK_MODE=false
+if [[ "$2" == "--sdk" ]]; then
+    SDK_MODE=true
+fi
+
+# Full directory path
+DIR="$BASE_DIR/$DIR_NAME"
+
+# Check if the directory exists
+if [ ! -d "$DIR" ]; then
+    echo "Directory does not exist: $DIR"
+    exit 1
+fi
+
+# Declare an array of filenames to check when in SDK mode
+declare -a SDK_FILES=("AddressManager" "L1CrossDomainMessengerProxy" "L1StandardBridgeProxy" "OptimismPortalProxy" "L2OutputOracleProxy")
+
+# Loop through each .json file in the directory
+for file in "$DIR"/*.json; do
```

Comment:
What is `$DIR` meant to be? `forge-artifacts` or `deployments`? We are planning on getting rid of `deployments` in favor of `superchain-ops`. It may be that `superchain-ops` is the right place for this script

Reply:
It's mean to be `deployments`. Mind if we just keep this in `deployments` for now? All of the docs use `deployments`

Reply:
Sure we can use this but we will need to coordinate the deletion of deployments bc the deployments pattern doesn't work well in the superchain context as well as a bunch of extra code required to maintain them